units/SDL3_imageFilter.inc

```diff
@@ -0,0 +1,122 @@
+{
+  This file is part of:
+
+    SDL3 for Pascal
+    (https://github.com/PascalGameDevelopment/SDL3-for-Pascal)
+    SPDX-License-Identifier: Zlib
+}
+
+//
+// All routines return:
+//   0   OK
+//  -1   Error (internal error, parameter error)
+//
+
+// SDL_imageFilterAdd: D = saturation255(S1 + S2)
+function SDL_imageFilterAdd(Src1, Src2, Dest: pcuint8; length: cuint): cint; cdecl;
```

Comment:
The following applies to all functions of the imageFilter inc file.

In accordance with ctypes unit, the following translation should be used 
char -> cchar
char * -> pcchar
unsigned char -> cuchar
unsigned char * -> pcuchar

I know, you always picked the right sized types by hand. Unfortunately, to be as tight as possible at the original code and as we decided to use ctypes, we should use the translations as mentioned above, I guess. Apart from this, the files looks great.